website/src/theme/ProtosaurusImage.tsx

```diff
@@ -28,7 +28,7 @@ export default function ProtosaurusImage() {
       const alt = element.dataset["imageAlt"];
 
       setModalInfo({
-        src,
+        src: require(`@site/static${src}`).default,
```

Comment:
This is so nice to have this kind of helper! Docusaurus is cool.

Reply:
indeed! :100: makes it very easy to deal with importing stuff.